apps/create-app.md

```diff
@@ -4,34 +4,46 @@ key: apps-create
 title: Create an app
 ---
 
-Once you have your test shop in place, you can create an app. You automatically receive a **Client ID**, **Client Secret** and **Access token** when you create an app in your test shop.
+Once you have your test shop in place, you can create an app.
+You automatically receive a **Client ID** and a **Client Secret** when you create an app in your test shop.
+After successful authorisation of your app, you will be assigned an **Access token**.
 
 | Value              | Description                                                                                             |
 |--------------------|---------------------------------------------------------------------------------------------------------|
 | **Client&nbsp;ID**      | The unique identifier for your app. |
-| **Client&nbsp;Secret**   | Used in combination with the **Client ID** for getting an **Access Token** during the [authorisation process](page:apps-install#authorisation-process).|
+| **Client&nbsp;Secret**   | Used in combination with the **Client ID** for getting an **Access token** during the [authorisation process](page:apps-install#authorisation-process).|
 | **Access&nbsp;token** | The token granted from your test shop to the newly created app. You can use this token for accessing the API. [Read here for further details.](page:apps-install) |
 
 # Get your credentials
 
-1. Within the administration area of your test shop, click the tab **Apps & Themes**. In the drop down menu, choose the field **Developer apps**.
+1. Within the administration area of your test shop, click the tab **Apps & Themes**.
+In the drop down menu, choose the field **Developer apps**.
     {% image click-apps.png %}{% endimage %}
 
 2. In the **Developer apps** view, click the button **Add**.
     {% image add-button.png %}{% endimage %}
 
-3. Fill out the entry field **App name**. This field is mandatory. Then click the button **Generate API access data**.
+3. Fill out the input field **App name**.
+Specify the **Application Callback URL**.
+Make sure the URL is https and publicly accessible.
```

Comment:
The callback URL can be HTTP for developer apps, but you can only submit an app with an HTTPS URL.